pkg/query/logical/analyzer.go

```diff
@@ -94,14 +95,17 @@ func (a *Analyzer) Analyze(_ context.Context, criteria *apiv1.QueryRequest, trac
 		useIndexScan := false
 		var fieldExprs []Expr
 		traceState := series.TraceStateDefault
+
 		for _, pairQuery := range criteria.GetFields() {
 			op := pairQuery.GetOp()
 			typedPair := pairQuery.GetCondition()
 			switch v := typedPair.GetTyped().(type) {
 			case *apiv1.TypedPair_StrPair:
+				// TODO: use metadata from Plugin?
```

Comment:
Retrieving the key from https://github.com/apache/skywalking-banyandb/blob/main/banyand/series/schema/sw/trace_series.textproto#L75

Reply:
Fixed. Extra functions are added to the Schema.